Resources/Prototypes/Entities/Objects/Tools/lantern.yml

```diff
@@ -60,10 +60,9 @@
         cell_slot: !type:ContainerSlot {}
 
 - type: entity
-  name: extra-bright lantern
```

Comment:
This item is a syndicate uplink item identified by its name. Personally I think the identifiable name and the old description makes it a bit more amusing for people to figure out what this is.